README.md

```diff
@@ -144,7 +144,7 @@ Go to your browser: http://127.0.0.1:8081
 ## Compiling and running with Docker
 
 The explorer can also be compiled using `docker build` as described below. By default it compiles
-against latest release (`release-v0.17`) branch of monero:
+against latest release tag (i.e. `v0.18.3.4`) branch of monero:
```

Comment:
```suggestion
against latest release tag (i.e. `v0.18.3.4`) of monero:
```

Reply:
The release tag is technically a branch here, but I'll allow it so people don't get confused 😅

Reply:
its not technically a branch. Branches arent tags and tags arent branches ;)